radio/src/gui/colorlcd/radio_hardware.cpp

```diff
@@ -202,5 +206,8 @@ void RadioHardwarePage::build(Window* window)
   new SetupButtonGroup(window, {0, 0, LCD_W - padding * 2, 0}, STR_DEBUG, BTN_COLS, PAD_ZERO, {
     {STR_ANALOGS_BTN, []() { new RadioAnalogsDiagsViewPageGroup(); }},
     {STR_KEYS_BTN, []() { new RadioKeyDiagsPage(); }},
+#if defined(FUNCTION_SWITCHES)
+    {STR_FS_BTN, []() { new RadioCustSwitchesDiagsPage(); }},  
```

Comment:
@philmoz Can you make it so buttons are big enough for two lines of text (again?)... stepping away from computer for a bit after the rebase...

Reply:
![screenshot_t15_24-06-07_16-52-47](https://github.com/EdgeTX/edgetx/assets/9474356/545cab83-ddb2-4706-a175-2e59a1e02fba)

Reply:
That'll do too! Thanks :)